truss-chains/truss_chains/definitions.py

```diff
@@ -365,10 +365,13 @@ def get_asset_spec(self) -> AssetSpec:
         return self.assets.get_spec()
 
 
+DEFAULT_TIMEOUT_SEC = 600
```

Comment:
this seems extremely high to me...

Reply:
True. This unfortunately factored in the "wake time" for a nested chain. We have improved chain wake up on the radar, but it's currently not there yet. Do you think it's safe to reduce the default, without causing unintended side effects for existing use cases at the moment?